src/FlaUI.Core/AutomationElements/TextBox.cs

```diff
@@ -11,6 +12,25 @@ namespace FlaUI.Core.AutomationElements
     /// </summary>
     public class TextBox : AutomationElement
     {
+        internal class Win32Constants
```

Comment:
There is a `WindowsAPI` namespace in FlaUI.Core, you should put that stuff in there

Reply:
Done.